src/testdom/testrules.ts

```diff
@@ -169,7 +173,53 @@ export function resolveTestRules(serviceClass: number) {
     return builtinTestRules[serviceClass]
 }
 
+function makeEventTest(test: ServiceMemberTestNode, event:string, flag: number) {
+    test.appendChild(
+        new EventTest(
+            `${event}`,
+            GamepadEvent.ButtonsChanged,
+            (node, logger) => {
+                const { event } = node
```

Comment:
Isn't there an event test node that does this already?

Reply:
yes, using it already.